src/server/auth/server/api_server.go

```diff
@@ -83,8 +83,13 @@ type apiServer struct {
 	watchesEnabled bool
 }
 
+type InternalAuthServer struct {
+	APIServer    authiface.APIServer
+	EnvBootstrap func(context.Context) error
+}
```

Comment:
This is really a lot of work to go to to make a method in `apiServer` public.  Would it be unreasonable to rename `apiServer` to `APIServer` and  `s.envBootstrap` to `s.EnvBootstrap` and skip this indirection?  (Now that I think about it, not strictly necessary to make apiServer public... but I hate constructors that return private types, because then you can only create variables that contain them with `:=`.)